### DISCUSSION TITLE
Dynamic configuration

### DISCUSSION THREAD

**Discussion by kotyara85** [ory/hydra#3513] [Q&A]
Hello everyone,
Does hydra support Dynamic Configuration for redirect_uris?
https://www.rfc-editor.org/rfc/rfc6749#section-3.1.2.3

We need to have some wildcards for subdomains
Thanks

**Comment by vinckr** [comment]
Hello @kotyara85 
No, Ory Hydra does not support wildcards in the redirect_uris. The redirect URL must be exactly matched and whitelisted in your client configuration. You can set multiple redirect URIs, see the example here: https://www.ory.sh/docs/hydra/guides/oauth2-clients#multiple-redirect-uris

See here also: https://github.com/ory/hydra/issues/3037#issuecomment-1072284524
